postgres/scripts/changeGeomType.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE projects ALTER COLUMN geom SET DATA TYPE geometry;
```

Comment:
this is important. We need to run this once we deploy the changes to the production postgres db.